kuznetsov/quizzes/src/main/java/com/kuznetsov/dao/impl/services/DataBaseAdapter.java

```diff
@@ -0,0 +1,9 @@
+package dao.impl.services;
+
+import java.sql.SQLException;
+
+public interface DataBaseAdapter {
+    int setToDB(String value) throws SQLException;
```

Comment:
Maybe setToDB is not right name for method. For example, looking in interface, I don't understand what this method do

Reply:
I fix bad manes